docs/upgrade/0.3.5-to-0.3.6.rst

```diff
@@ -0,0 +1,112 @@
+Upgrade from 0.3.5 to 0.3.6
+============================
+
+SecureDrop 0.3.6 was a maintenance release to update the expiration
+date on the GPG signing key used for the Freedom of the Press
+apt repository. The signing key expired on October 26, 2015, preventing
+automatic updates from Freedom of the Press until the issue is
+manually resolved.
+
+
+.. note:: We're considering creating a separate ``securedrop-keyring`` 
```

Comment:
Did this during the hackathon, so maybe you could think about removing this note. I hadn't thought of naming it `securedrop-keyring`, I went with Tor's convention instead. Maybe `securedrop-keyring` is better.

https://github.com/freedomofpress/apt.freedom.press-keyring